### PR TITLE
Feature/15 support new earmark structure

### LINKED ISSUE #15: Round 11 - Update DAOBot to be able to support new Earmark structure
Various things changed since last round:
- 2 new earmarks were added
- Airtable/Calculations need to be updated to support these changes

New earmark structure will require DAOBot funding calculations to be updated.
https://oceandao.notion.site/Round-11-Votes-fb8c4538405e4f10bcbcaa7c10bbf060

DoD:
- [ ] Setup DAOBot to be configurable so we can use previous earmarks & future earmarks.
- [ ] Setup Airtable columns as required to be able to support historical and new earmarks.
- [ ] Test & Verify DAOBot provides coverage for historical and new earmarks.
- [ ] Deploy latest version of DAOBot.

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #15.

### CODE REVIEW

File: src/test/airtable/test_airtable_round_winners.test.js [hunk 1]
```diff
@@ -240,4 +249,31 @@ describe('Calculating Winners', function() {
...
+
+        allProposals[5].fields['USD Granted'] = 2000
+
+        //filter all proposals that have Earmarks
+        let proposalsWithEarmark = allProposals.filter(p => p.get('Earmarks') !== undefined)
+
+        //calculate and get all winning proposals
+        let earmarkedResults = calculateWinningAllProposals(proposalsWithEarmark, fundingRound, oceanPrice)
+        should.equal(earmarkedResults.winningProposals.length, 5)
+    });
```
Comment (idiom-bytes) [inline src/test/airtable/test_airtable_round_winners.test.js:278]: Thank you for creating this new test Norbert. It really helps to review the commit, and to double check that the design is working as expected. I have reviewed the rest of the code lightly, and am basing my review mostly from this, which leads me to believe it's working as intended.

File: src/scripts/update_funding_round.js [hunk 2]
```diff
@@ -111,25 +111,50 @@ const main = async () => {
...
                     'Proposals': allProposals.length,
                     'OCEAN Price': tokenPrice,
-                    'Max Grant': maxGrantOCEAN,
-                    'Earmarks': earmarkedStructureWithOCEAN,
-                    'Funding Available': fundingAvailableOCEAN,
+                    'Max Grant': maxGrant,
+                    'Earmarks': JSON.stringify(completeEarstructuresValues(curRound, tokenPrice, basisCurrency)),
+                    'Funding Available': fundingAvailable,
+                    'Max Grant USD': maxGrantUSD,
+                    'Earmarked USD': earmarkedUSD,
```
Comment (claudiaHash) [inline src/scripts/update_funding_round.js:156]: I think you should make some changes here, regarding the `Earmarked USD`

File: src/scripts/update_funding_round.js [hunk 3]
```diff
@@ -175,6 +200,6 @@ const main = async () => {
     }
 }
 
-module.exports = {addOCEANValueToEarmarks};
+module.exports = {completeEarstructuresValues};
```
Comment (claudiaHash) [inline src/scripts/update_funding_round.js:203]: Not sure we need to export this function here

File: src/scripts/update_funding_round.js (hunk 2 above)
Comment (KatunaNorbert) [reply src/scripts/update_funding_round.js:156]: yes, that should be removed, forgot to remove it when getting the new changes from main

File: src/scripts/update_funding_round.js (hunk 3 above)
Comment (KatunaNorbert) [reply src/scripts/update_funding_round.js:203]: Right, I'll remove it, thanks

**Review by AnaLoznianu — APPROVED** [review]
Great work!

**Pull request merged**